Makefile

```diff
@@ -180,7 +180,9 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
 
 check-generate: generate
-	@test -z $(shell git status --untracked-files=no --porcelain)
+	@if [ $(shell git status --untracked-files=no --porcelain | wc -l ) != 0 ]; then \
```

Comment:
While you're at it, maybe fix `check-fmt` too?

Reply:
Thanks for reminding! done